### PR TITLE
Feature/search bar - Added new search styling and submit functionality

### PR DESCRIPTION
**Pull request by IncredibleMrTim**
### What

Added search styles and filter submit

### How to review
- With the search controller enabled 
- do a search
- use the filters to the left hand side
- the results of the search should be filtered

### Who can review

anyone

### CODE REVIEW

File: src/scss/dp/overrides/components/_search.scss [hunk 1]
```diff
@@ -149,19 +165,53 @@
...
-      width: 52px;
-      margin-top: -2px;
-      float: left;
-    }
+.search-results {
+
+}
+
+.search-summary {
+  width: 100% !important;
```
Comment (AnnaEmson) [inline src/scss/dp/overrides/components/_search.scss:187]: `!important` should not be used only if you are overwriting the library in this situation I think is not a case

File: src/scss/dp/overrides/components/_search.scss [hunk 2]
```diff
@@ -149,19 +165,53 @@
       height: ($baseline * 5);
       padding-top: ($baseline / 2);
     }
+    //
+    //&--body {
```
Comment (AnnaEmson) [inline src/scss/dp/overrides/components/_search.scss:169]: remove it?

File: src/scss/dp/overrides/components/_search.scss [hunk 3]
```diff
@@ -9,8 +9,16 @@
     height: 80px;
   }
 
+  &--col-border-bottom {
+    border-bottom: 1px solid #000;
+  }
+
+  &--results-text {
+    font-size: 200px;
```
Comment (AnnaEmson) [inline src/scss/dp/overrides/components/_search.scss:17]: `font-size: 200px;` is it a typo?

File: src/scss/dp/overrides/components/_search.scss [hunk 4]
```diff
@@ -9,8 +9,16 @@
     height: 80px;
   }
 
+  &--col-border-bottom {
+    border-bottom: 1px solid #000;
```
Comment (AnnaEmson) [inline src/scss/dp/overrides/components/_search.scss:13]: not sure about this app but those usually should be a variables?

File: src/js/dp/search/__test__/search.test.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+it('should work', () => {
```
Comment (AnnaEmson) [inline src/js/dp/search/__test__/search.test.js:1]: hmmm.. this looks like copied example  .. can you fill it in with proper test

File: src/js/dp/search/__test__/search.test.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
...
+    expect(true).toBeTruthy()
+})
```
Comment (AnnaEmson) [inline src/js/dp/search/__test__/search.test.js:3]: new line as suggested?

File: src/scss/dp/overrides/components/_search.scss [hunk 7]
```diff
@@ -149,19 +165,49 @@
...
-      width: 52px;
-      margin-top: -2px;
-      float: left;
-    }
+.search-summary {
+  width: 100% !important;
+  padding: 36px 0 46px;
+
+  .base-font {
+    font-size: 30px;
```
Comment (AnnaEmson) [inline src/scss/dp/overrides/components/_search.scss:187]: should you be using rem or em as font values for the website to meet accessibility requirements?  (sorry if it is not relevant have no clue what this app is for)

File: src/scss/dp/overrides/components/_search.scss [hunk 8]
```diff
@@ -53,13 +62,14 @@
     font-family: $base-font-family;
     font-weight: inherit;
     line-height: $base-line-height;
-    font-size: 17px;
+    font-size: 18px;
```
Comment (AnnaEmson) [inline src/scss/dp/overrides/components/_search.scss:65]: here the same ... consider using relative font size value for accessibility ?

File: src/scss/dp/overrides/components/_search.scss [hunk 9]
```diff
@@ -149,19 +165,53 @@
...
+}
 
-    &--body {
-      height: 52px;
-      width: 52px;
-      margin-top: -2px;
-      float: left;
-    }
+.search-results {
+
```
[the remark this answers is not in the document]
Comment (IncredibleMrTim) [reply src/scss/dp/overrides/components/_search.scss]: fixed

File: src/js/dp/search.js [hunk 10]
```diff
@@ -0,0 +1,14 @@
+const submitFilterForm = () => {
+  console.log('T1', frm)
```
Comment (AnnaEmson) [inline src/js/dp/search.js:2]: console.log?

File: src/js/dp/search/__test__/search.test.js (hunk 5 above)
Comment (AnnaEmson) [reply src/js/dp/search/__test__/search.test.js]: I don't think we need this empty test at all

File: src/scss/dp/overrides/components/_search.scss (hunk 7 above)
Comment (IncredibleMrTim) [reply src/scss/dp/overrides/components/_search.scss]: fixed

File: src/scss/dp/overrides/components/_search.scss [hunk 11]
```diff
@@ -32,6 +40,7 @@
     font-family: $base-font-family;
     font-weight: inherit;
     line-height: $base-line-height;
+    width: 100%;
```
[the remark this answers is not in the document]
Comment (IncredibleMrTim) [reply src/scss/dp/overrides/components/_search.scss:83]: fixed

File: src/scss/dp/overrides/components/_search.scss (hunk 8 above)
Comment (IncredibleMrTim) [reply src/scss/dp/overrides/components/_search.scss:105]: fixed

File: src/js/dp/search.js (hunk 10 above)
Comment (IncredibleMrTim) [reply src/js/dp/search.js:2]: fixed

File: src/js/dp/search/__test__/search.test.js (hunk 6 above)
Comment (IncredibleMrTim) [reply src/js/dp/search/__test__/search.test.js]: file removed

File: src/scss/dp/overrides/components/_search.scss (hunk 4 above)
Comment (IncredibleMrTim) [reply src/scss/dp/overrides/components/_search.scss]: fixed

**Review by AnnaEmson — APPROVED** [review]

**Pull request merged**